### ISSUE TITLE
There is a bug with one of the dependencies that was updated on November 25, 2020

### ISSUE THREAD

**Issue by 0DUDDU** [kreait/firebase-tokens-php#32]
**Describe the bug**
I ran composer update, "lcobucci/jwt" has been updated from 3.3.3 to 3.4.0, then authentication doesn't work anymore

**To Reproduce**
install fresh or update current install and try to verify a token

```php
app('firebase.auth')->verifyIdToken($request->get("jwt"));
```

**Expected behavior**
it should auth correctly

**Environment (please complete the following information):**
 - OS: Windows 10
 - PHP version: 7.4.0
 - kreait/firebase-tokens Version: 1.11.0
 - firebase/php-jwt Version : v5.2.0 
 - kreait/firebase-php Version: 5.11.0

**Additional context**
Error message 
```
The token has an invalid signature: Implicit conversion of keys from strings is deprecated. Please use InMemory or LocalFileReference classes.
```

Stacktrace
```
\vendor\kreait\firebase-tokens\src\Firebase\Auth\Token\Verifier.php:145
\vendor\kreait\firebase-tokens\src\Firebase\Auth\Token\Verifier.php:68
\vendor\kreait\firebase-php\src\Firebase\Auth\IdTokenVerifier.php:53
\vendor\kreait\firebase-php\src\Firebase\Auth.php:581
```

**Comment by jeromegamez** [comment]
I'll try to create a new release in the next few days. In the meantime please see https://github.com/lcobucci/jwt/issues/550 - there are some approaches in the thread on how to handle the deprecation messages that explain them better than I could :)

**Comment by jeromegamez** [comment]
Fixed with release 1.12 (already since last night, but I forgot to notify you here) 😅

Let me know if there are problems that I missed 🤞

**Comment by 0DUDDU** [comment]
Thanks, it work well 😃
